client/components/Application/Dialogs/WelcomeBanner.tsx

```diff
@@ -8,14 +8,21 @@ import * as store from '@client/store'
 import * as React from 'react'
 import welcomescreenImg from './assets/welcomescreen.png'
 import { styled } from '@mui/material/styles'
+import { saveConfig } from '../../../store/actions/config'
+
+export default async function WelcomeBanner() {
+
+  const config = store.useStore((state) => state.config)
```

Comment:
@roll I am getting an error when reading the config value from the store. Is it because it is not set? Am i doing something wrong? In general I need help with this commit.

Reply:
@guergana 
Config is being loaded on the application startup so I think you need to wait till it's available

Reply:
@pdelboca 
@guergana 
Because config is needed during the initialization it might need other storage, for example, `localStorage`. Or probably we can use Zustand persistence - https://github.com/okfn/opendataeditor/blob/main/client/store/store.ts#L31-L32

Reply:
Hi @roll  I don't understand why the config.json solution wouldn't be enough. Is it not loaded on initialization? I am more than happy to use localStorage and keep this in the frontend. But I thought we might want to have only one way of persisting the data? In that case should we move the other variable to localStorage as well?